Resources/Prototypes/DeltaV/Roles/Jobs/Legal/chief_justice.yml

```diff
@@ -0,0 +1,53 @@
+- type: job
+  id: ChiefJustice
+  name: job-name-chiefjustice
+  description: job-description-chiefjustice
+  playTimeTracker: JobChiefJustice
+  antagAdvantage: 6 # DeltaV - Reduced TC: Head of Staff
+  requirements:
+#    - !type:RoleTimeRequirement
+#      role: JobClerk
+#      time: 36000 # DeltaV - 10 hours
+# Commenting out the clerk requirement for now until clerk has been around long enough for people to actually get the hours required.  
```

Comment:
I feel this would encourage people to go straight to Chief Justice instead of Clerk, thereby disqualifying a sizable amount of people by the time this is uncommented.
If Clerk can substitute the Chief Justice in all instances, I see no reason why we wouldn't just uncomment this and let people naturally gain the time requirements. It would also save us a PR, which is always nice. Of course, call it out if I'm missing anything.

Reply:
that's true, I'll probably change that then. I'll be honest, this is like the most difficult and confusing thing I've ever attempted to add to the game so I'll take any advice and help I can get.